tests/model_selection/test_cross_validate.py

```diff
@@ -209,7 +209,7 @@ def test_happy_path(
             dataset=self.dataset,
             splitter=splitter,
             metrics=self.metrics,
-            models=self.models,
+            models=self.models,  # type: ignore
```

Comment:
We should anyway remove this ignoring comment. And that one below as well

Reply:
Removed ignoring comment, added type definition for self.models.